src/NuGet.Core/NuGet.Packaging.Extraction/NuGet.Packaging/Strings.Designer.cs

```diff
@@ -39,13 +39,13 @@ internal class Strings {
         internal static global::System.Resources.ResourceManager ResourceManager {
             get {
                 if (object.ReferenceEquals(resourceMan, null)) {
-                    global::System.Resources.ResourceManager temp = new global::System.Resources.ResourceManager("NuGet.Packaging.Strings", typeof(Strings).Assembly);
+                    global::System.Resources.ResourceManager temp = new global::System.Resources.ResourceManager("NuGet.Packaging.Extraction.NuGet.Packaging.Strings", typeof(Strings).Assembly);
```

Comment:
This is effectively the fix, duplicated across the other `.Designer.cs` files. Everything else is "boilerplate" or tidying up.